versioned_docs/version-2.7/azure/partials/common/_prerequisites.mdx

```diff
@@ -0,0 +1,18 @@
+### Azure Prerequisites
+
+For kubefirst to be able to provision your Azure cloud resources:
```

Comment:
```suggestion
For Kubefirst to be able to provision your Azure cloud resources:
```

Reply:
Anything under the `versioned_docs` folder is autogenerated by the `/docs` folder